backend/app/models/group.rb

```diff
@@ -0,0 +1,6 @@
+# frozen_string_literal: true
+
+class Group < ApplicationRecord
+  belongs_to :user
```

Comment:
Let's name `user` as `author` here, because we will have students, also attached to the group and we will need to determine who is who:)

Expected result:

```ruby
Group.last.author #=> <User #123 ...>
```